extensions/amp-access-laterpay/0.1/laterpay-impl.js

```diff
@@ -219,12 +219,13 @@ export class LaterpayVendor {
     const urlPromise = this.accessService_.buildUrl(
       url, /* useAuthData */ false);
     return urlPromise.then(url => {
-      dev().fine(TAG, 'Authorization URL: ', url);
+      return this.accessService_.buildLoginUrl(url);
+    }).then(url => {
+      dev().info(TAG, 'Authorization URL: ', url);
       return this.timer_.timeoutPromise(
           AUTHORIZATION_TIMEOUT,
           this.xhr_.fetchJson(url, {
             credentials: 'include',
-            requireAmpResponseSourceOrigin: true,
```

Comment:
Are you sure we need to remove it? Usually this is a somewhat important piece of data, e.g. when more than one origin is supported by the same authorization services.